server/models/Requests.js

```diff
@@ -36,5 +37,6 @@ const requestSchema = new mongoose.Schema({
     default: false,
   },
 });
+requestSchema.plugin(AutoIncrement, {id: 'request_seq', inc_field: 'requestId'});
```

Comment:
What is the purpose here? Couldn't we just use the Mongo ID attached to the model?

Reply:
Yes, I will update this to use the objects _id field!